apps/atomy-q/API/tests/Feature/QuoteIngestionPipelineTest.php

```
@@ -101,6 +111,39 @@ private function createRfq(User $user): Rfq
+        $config = require config_path('atomy.php');
+
+        if ($previousMode === false) {
+            putenv('QUOTE_INTELLIGENCE_MODE');
+        } else {
+            putenv('QUOTE_INTELLIGENCE_MODE=' . $previousMode);
+            $_ENV['QUOTE_INTELLIGENCE_MODE'] = $previousMode;
+            $_SERVER['QUOTE_INTELLIGENCE_MODE'] = $previousMode;
+        }
+
+        return $config;
+    }
```

PHPMD (2.15.0)

[error] loadAtomyConfigContractWithoutQuoteIntelligenceMode accesses the super-global variable $_ENV. (undefined)

(Superglobals)